src/Clap-Core.package/ClapAlternative.class/README.md

```diff
@@ -0,0 +1,19 @@
+I am a class which represents all the possible ways to execute a command.
```

Comment:
This is wrong. The alternative is used when there are several ways to recognize a single parameter. If you had to enumerate every single combination of parameters for all possible commands as you suggest below, it wouldn't be very practical…